client/src/components/Sidebar/common/Icon.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+export interface IconProps {
+  name: string;
+  className?: string;
+  title?: string;
+}
+
+/**
+ * Provide a `title` as an accessible label intended for screen readers.
+ */
+const Icon: React.FunctionComponent<IconProps> = ({ name, className, title }) => (
+  <>
+    <svg className={`icon icon-${name} ${className || ''}`} aria-hidden="true">
+      <use href={`#icon-${name}`}></use>
+    </svg>
+    {title ? (
```

Comment:
You can avoid a ternary here
```suggestion
    {title && ( <span>...</span> )}
```